data/src/main/java/com/globant/equattrocchio/data/ImagesRepository.java

```diff
@@ -0,0 +1,36 @@
+package com.globant.equattrocchio.data;
+
+import com.globant.equattrocchio.data.entities.ImageEntity;
+import com.globant.equattrocchio.domain.enities.Image;
+
+import java.util.List;
+
+import io.realm.Realm;
+
+
+public class ImagesRepository {
```

Comment:
You need create an interface to manipulate this class

Reply:
done